nexus/reconfigurator/planning/src/mgs_updates/mod.rs

```diff
@@ -1731,42 +1240,57 @@ mod test {
         logctx.cleanup_successful();
     }
 
-    // Updates a whole system's SPs at once
+    // Updates as much of a whole system at once as we can
     #[test]
     fn test_whole_system_simultaneous() {
```

Comment:
Maybe we should change the name of the test at this point? Something like `test_whole_system_simultaneous_per_component` or something similar?